EngineeringModel.Tests/ViewModels/FiniteStateBrowser/FiniteStateBrowserViewModelTestFixture.cs

```diff
@@ -192,8 +192,17 @@ public void VerifyThatContextMenuPopulated()
             this.iteration.ActualFiniteStateList.Add(actualList);
 
             var viewmodel = new FiniteStateBrowserViewModel(this.iteration, this.session.Object, this.thingDialogNavigationService.Object, this.panelNavigationService.Object, null, null);
-            
-            // no row selected
+            viewmodel.FiniteStateList.Add(new CDP4Composition.FolderRowViewModel("Actual List", "Possible Finite State List", viewmodel.Session, viewmodel));
```

Comment:
Adding viewmodels is not necessary. It is done in the Initialize method.

Reply:
That is correct. Code removed.